debian_packages.yaml

```diff
@@ -40,15 +40,38 @@
     - "libgomp1"
     - "libstdc++6"
 
-# python and java only build on a subset of architectures
+# debian 11, limited architectures, java only
+- distros: ["debian11"]
+  archs: ["amd64", "arm64", "s390x"]
+  packages:
+    - "fontconfig-config"
+    - "fonts-dejavu-core"
+    - "libbrotli1"
+    - "libcrypt1" # TODO: glibc library for -lcrypt; maybe should be in cc?
+    - "libexpat1"
+    - "libfontconfig1"
+    - "libfreetype6"
+    - "libglib2.0-0"
+    - "libgraphite2-3"
+    - "libharfbuzz0b"
+    - "libjpeg62-turbo"
+    - "liblcms2-2"
+    - "libpcre3"
+    - "libpng16-16"
+    - "libuuid1"
+    - "openjdk-11-jdk-headless"
+    - "openjdk-11-jre-headless"
+    - "openjdk-17-jdk-headless"
+    - "openjdk-17-jre-headless"
+    - "zlib1g"
+
+# debian 11, limited architectures, python only
 - distros: ["debian11"]
   archs: ["amd64","arm64"]
   packages:
-  # python
     - "dash"
     - "libbz2-1.0"
     - "libcom-err2"
-    - "libcrypt1"  # TODO: glibc library for -lcrypt; maybe should be in cc?
```

Comment:
I think it's okay to leave this in the python one, duplicates are handled by the system.

Reply:
Sure, I've re-added the libcrypt1 dependency to the list of Python dependencies in the latest commit.